etc/terraform-arm64.conf

```
@@ -0,0 +1,33 @@
+# target architecture - i386, amd64 or all
+ARCH="arm64"
+
+# base codename
+BASECODENAME="noble"
+
+# base version
+BASEVERSION="24.04"
+
+# distribution codename
+CODENAME="circe"
+
+# distribution version
+VERSION="8.0"
+
+# distribution channel
+CHANNEL="stable"
+
+# distribution name
+NAME="elementary OS"
+
+# mirror to fetch packages from
+MIRROR_URL="http://ports.ubuntu.com/ubuntu-ports/"
+
+# use HWE kernel and packages?
+HWE_KERNEL="yes"
```

```suggestion
HWE_KERNEL="no"
```

Based on etc/terraform-daily-8.0-azure.conf

---

> Could you clarify the reasoning behind this?

This looks like because the daily conf was originally built on Mantic, which was not a LTS: #690. After that the daily conf was rebased on Noble but we seemed to forget updating this parameter.

So, I agree with you to use the hwe kernel here. I'm sorry for my wrong suggestion.